README.md

```diff
@@ -355,6 +355,8 @@ Table of Contents
 * [Appachhi](https://appachhi.com/) :moneybag:
 * [AppSpector](https://appspector.com/) :moneybag:
 * [Perfetto](https://perfetto.dev/)
+* [Smartlook](https://www.smartlook.com/)
```

Comment:
Thank you very much for the PR, could you add `:moneybag:`, please? :)

Reply:
done... cheers. :)